### ISSUE TITLE
SEQIOUA 15.0

### ISSUE THREAD

**Issue by minaWesam** [valnoxy/t480-oc#159]
has anyone been able to boot in 15.0 i have been able after disabling bluetooth and added -V in boot args but i cant disable the verbose and fix bluetooth etc and also sleep and shutdown and reset doesnt work at all

**Comment by rpcameron** [comment]
I am writing this from a T580 using the 1.4.0-HeliPort EFI and currently running Sequoia 15.0.

(As a previous commenter noted, iServices do not seem to working with AirPortItlwm. I think they stopped working with 14.4, when the secure boot settings had to be disabled. However, both under 14.x and now 15.0, iServices are working with HeliPort.)

**Comment by minaWesam** [comment]
does sleep work and restart and shut down

**Comment by rpcameron** [comment]
Yep, everything working as expected. There is the occasional reboot upon shutdown (but as mentioned on another issue, this is likely due to USB mapping issues, and I am too lazy to make a Windows install just to fix that). Wake from sleep is fine.

The only big issue is that HeliPort makes the system think the wireless card is a second ethernet port, so things that require wifi (AirDrop, iPhone Mirroring, etc.) won't work, because macOS thinks there is no wifi.

**Comment by rpcameron** [comment]
Also to note: Because wifi is via HeliPort and not itlwm, bluetooth is also missing on Sequoia. (Meaning, it sees that there is BT, but that it would see any devices.)

**Comment by lolipuru** [comment]
i saw some discussion for making airportltwm works by spoofing it to broadcom wifi card, then patch with OCLP

i already replace my wifi card with BCM94360CS2, but i can try with my intel card

**Comment by lolipuru** [comment]
alright, i got wifi working with airportltwm (ventura), using OCLP patch (by spoof wifi card as broadcom so OCLP root patch will be able to applied the patch)

![image](https://github.com/user-attachments/assets/593cee65-0672-4426-a638-bd90a42111e7)

but this fix only applied to wifi only, bluetooth still not working (or maybe i just using outdated bluetooth kext)

**Comment by XD-BD** [comment]
im trying to create a boot-loader pendrive with opencore for macOS 15 sequoia but i dont have an ethernet port can  do it through wifi
